shared-module/src/locales/en/course-material.json

```diff
@@ -126,5 +126,7 @@
   "help-text-exercise-involves-peer-review": "To complete this exercise, you will need to peer review {{peer_reviews_to_give}} answers from other students after submitting your own answer. After that, you'll receive peer reviews from other students before getting the points.",
   "waiting-for-peer-reviews-explanation": "Other students are peer reviewing your answer. Please come back later to see the results.",
   "help-text-answer-has-been-reviewed-and-locked": "Your answer has been reviewed and graded. New submissions are no longer allowed.",
-  "additional-module": "Additional module"
+  "additional-module": "Additional module",
+  "page": "PAGE",
```

Comment:
In translations we should write strings normally and then in css `text-transform: uppercase;`